dagoba/dagoba.markdown

```diff
@@ -1,18 +1,20 @@
-# Dagoba: an in-memory graph database
+# Dagoba: an in-memory graph database [^titlefoot]
 
-> "When we try to pick out anything by itself we find that it is bound fast by a thousand invisible cords that cannot be broken, to everything in the universe." 
-> --- John Muir 
+[^titlefoot] This database started life as a library for managing Directed Acyclic Graphs, or DAGs. It was originally intended to come with a silent 'h' at the end, an homage to the swampy fictional planet, but reading the back of a chocolate bar one day we discovered the sans-h version refers to a place for silently contemplating the connections between things, which seems even more fitting.
```

Comment:
I'm going to name the database "Dagoba" in the chapter text, and move this footnoote reference from the title down into the chapter text itself. I don't trust our rendering process to handle this well.